test/LiveDeploy.t.sol

```diff
@@ -0,0 +1,359 @@
+            _testLZDepositAndBridge(ERC20(mainConfig.base), amount);
+        } else { }
+    }
+
+    function testDepositBaseAssetAndUpdateRate(uint256 depositAmount, uint96 rateChange) public {
+        AccountantWithRateProviders accountant = AccountantWithRateProviders(mainConfig.accountant);
+        // manual bounding done because bound() doesn't exist for uint96
+        rateChange = rateChange % uint96(mainConfig.allowedExchangeRateChangeUpper - 1);
+        rateChange = (rateChange < mainConfig.allowedExchangeRateChangeLower + 1)
+            ? mainConfig.allowedExchangeRateChangeLower + 1
+            : rateChange;
+
+        // mint a bunch of extra tokens to the vault for if rate increased
+        deal(mainConfig.base, mainConfig.boringVault, depositAmount);
+
+        depositAmount = bound(depositAmount, 1, 10_000e18);
+
+        _depositAssetWithApprove(ERC20(mainConfig.base), depositAmount);
+        BoringVault boringVault = BoringVault(payable(mainConfig.boringVault));
+        uint256 expected_shares = depositAmount;
+        assertEq(
+            boringVault.balanceOf(address(this)),
+            expected_shares,
+            "Should have received expected shares 1:1 for base asset"
+        );
+
+        // update the rate
+        vm.startPrank(mainConfig.exchangeRateBot);
+        uint96 newRate = uint96(accountant.getRate()) * rateChange / 10_000;
+        accountant.updateExchangeRate(newRate);
+        vm.stopPrank();
+
+        uint256 expectedAssetsBack = depositAmount * rateChange / 10_000;
+
+        // attempt a withdrawal after
+        TellerWithMultiAssetSupport(mainConfig.teller).bulkWithdraw(
+            ERC20(mainConfig.base), expected_shares, expectedAssetsBack, address(this)
+        );
+        assertEq(
+            ERC20(mainConfig.base).balanceOf(address(this)),
+            expectedAssetsBack,
+            "Should have been able to withdraw back the depositAmount with rate factored"
+        );
+    }
+
+    function testDepositBaseAsset(uint256 depositAmount) public {
+        depositAmount = bound(depositAmount, 1, 10_000e18);
+        _depositAssetWithApprove(ERC20(mainConfig.base), depositAmount);
+
+        BoringVault boringVault = BoringVault(payable(mainConfig.boringVault));
+        uint256 expected_shares = depositAmount;
+        assertEq(
+            boringVault.balanceOf(address(this)),
+            expected_shares,
+            "Should have received expected shares 1:1 for base asset"
+        );
+
+        // attempt a withdrawal after
+        TellerWithMultiAssetSupport(mainConfig.teller).bulkWithdraw(
+            ERC20(mainConfig.base), expected_shares, depositAmount, address(this)
+        );
+        assertEq(
+            ERC20(mainConfig.base).balanceOf(address(this)),
+            depositAmount,
+            "Should have been able to withdraw back the depositAmount"
+        );
+    }
```

Comment:
Let's do this basic base asset deposit, but also with a fuzzed exchangeRate. 
So unlike the other test here which compares how much they get out after an exchange rate chagne, this would just be initial deposit and withdrawals on the fuzzed exchangerate, with no changes in the exchange rate in between (but the starting exchange rate would not be 1).